libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount.h

```diff
@@ -109,19 +109,10 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, readonly, getter = isUserDeleted) BOOL userDeleted;
```

Comment:
Ping @trooper2013 : just a reminder that the guest user account functionality ([Line 101 et al](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1939/files#diff-e0241fd372a95e31ff1171dd90b65146R101)) should be removed from `SFUserAccount`.